software/sys_controller/ossc/av_controller.c

```diff
@@ -303,7 +303,8 @@ status_t get_status(tvp_input_t input, video_format format)
             status = (status < MODE_CHANGE) ? MODE_CHANGE : status;
 
         if (update_cur_vm) {
-            tvp_setup_hpll(cm.sample_mult*video_modes[cm.id].h_total, clkcnt, cm.cc.tvp_hpll2x && (video_modes[cm.id].flags & MODE_PLLDIVBY2));
+            cm.h_mult_total = (video_modes[cm.id].h_total*cm.sample_mult) + video_modes[cm.id].h_total_adj;
```

Comment:
It doesn't seem that would work. N64 needs 320x240 H.samplerate 386 + H.total_adj 3 on LineX4 and H.total_adj 4 on LineX5.

If set to 4, H.total_adj % cm.sample_mult would result in 0 on LineX4 and 4 in LineX5.

Reply:
Then it might be better to saturate fine-tune adjustment to cm.sample_mult-1:
`cm.h_mult_total = (video_modes[cm.id].h_total*cm.sample_mult) + ((video_modes[cm.id].h_total_adj < cm.sample_mult) ? video_modes[cm.id].h_total_adj : cm.sample_mult-1);`
That way it's not possible to accidentally increase base samplerate (e.g. from 386 to 387) by playing around with the fine-tune.

Reply:
Oh now I realized what you wanted to do.  Then your first suggestion is fine. I will send an update.